$OpenBSD: patch-src_dhcpcd_conf_5_in,v 1.1 2018/02/06 10:11:34 sthen Exp $

From 8f483d192082a953dd035f38ee4555735106f1fc Mon Sep 17 00:00:00 2001
From: Roy Marples <roy@marples.name>
Date: Fri, 2 Feb 2018 11:24:58 +0000
Subject: inet6: simplify setup of kernel

Rename if_checkipv6 to if_setup_inet6 so it's more descriptive.
simpify code as we don't really care about the result.
Restore the prior behavior of not disabling kernel RA handling
if dhcpcd is not doing it.

Index: src/dhcpcd.conf.5.in
--- src/dhcpcd.conf.5.in.orig
+++ src/dhcpcd.conf.5.in
@@ -22,7 +22,7 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd January 8, 2018
+.Dd February 2, 2018
 .Dt DHCPCD.CONF 5
 .Os
 .Sh NAME
@@ -480,9 +480,9 @@ See
 .%T "RFC 3927"
 .Re
 .It Ic noipv6
-Don't attempt to configure an IPv6 address.
+Don't solicit or accept IPv6 Router Advertisements and DHCPv6.
 .It Ic noipv6rs
-Disable solicitation and receipt of IPv6 Router Advertisements.
+Don't solicit or accept IPv6 Router Advertisements.
 .It Ic nolink
 Don't receive link messages about carrier status.
 You should only set this for buggy interface drivers.
